reactioncommerce/reaction
# Fix #3519 handle invalid card details with Authorize.net

Fixes #3519

### To test
1. Login as admin and enable shipping and Authorize.net
1. Try to checkout but for credit card number put "12345"
1. Observe you get normal field validation error.
1. Try credit card number "4040404040404040" and click "Complete your order"
1. Observe that you get `Error: Invalid card details [400]`
